### ISSUE TITLE
Add Support for Secure Pairing

### ISSUE THREAD

**Issue by qstambaugh** [azymohliad/watchmate#56]
Infinitime supports secure pairing. It would help prevent potential hijacking or MITM attacks

https://github.com/InfiniTimeOrg/InfiniTime/pull/796 
https://github.com/InfiniTimeOrg/InfiniTime/pull/886

Thank you for the great application! I've been using it reliably with my pinephone

**Comment by azymohliad** [comment]
Yeah, that's definitely a needed feature to have! I'll add it. Not sure when though, hopefully I'll dedicate some time to Watchmate in February, but no promises.
